tools/ci_build/github/azure-pipelines/orttraining-linux-nightly-gpu-on-device-training.yml

```diff
@@ -13,10 +13,10 @@ jobs:
 
   # Entry point for all on device training tests
   - script: |
-      docker run \
+      docker run -it \
```

Comment:
You should not put "-it" in a pipeline. it means interactive.

Reply:
I'm just testing for now, ignore all the changes to this PR until I ask for a re review.
Thanks